lightning/src/routing/gossip.rs

```diff
@@ -2380,7 +2366,9 @@ mod tests {
 		assert!(!network_graph.read_only().nodes().is_empty());
 		assert!(!network_graph.read_only().channels().is_empty());
 		network_graph.write(&mut w).unwrap();
-		assert!(<NetworkGraph>::read(&mut io::Cursor::new(&w.0)).unwrap() == network_graph);
+
+		let logger = Arc::new(test_utils::TestLogger::new());
```

Comment:
It's actually needed here because of the later assertion against the `NetworkGraph<Arg<test_utils::TestLogger>>` returned by `create_network_graph`. I guess using `Arc` was a convenience to avoid passing or returning a `TestLogger` to/from that function, though an `Rc` would have sufficed.